src/libmongoc/src/mongoc/mongoc-topology-description.c

```diff
@@ -676,10 +678,10 @@ _check_any_server_less_than_wire_version_v5_0 (const void *sd_,
  * secondary server for using aggregate pipelines that contain writing stages
  * (i.e. '$out' and '$merge').
  */
-static mongoc_read_mode_t
-_calc_effective_read_mode (const mongoc_topology_description_t *td,
-                           mongoc_ss_optype_t optype,
-                           mongoc_read_mode_t requested_read_mode)
+static bool
+_force_read_primary (const mongoc_topology_description_t *td,
```

Comment:
```suggestion
_must_use_primary (const mongoc_topology_description_t *td,
```
Consistency with name of data member.